config/default.yml

```diff
@@ -13,3 +13,6 @@ server:
 grafana:
   labels:
     app: 'boilerplate-mern'
+  
+inspectlet:
+  wid: '812310448'
```

Comment:
By adding this in default, you are suggesting that each app that is built from the boilerplate would have this value. This is not true.

Please remove it from here.